### PR TITLE
Sematext Agent docs

### PR DESCRIPTION
**Pull request by rabbitstack**
Reopened PR to merge into `develop` branch.

### CODE REVIEW

**Comment by rabbitstack** [comment]
@otisg `sematext/agent:latest` is actually the stable image version.

**Comment by otisg** [comment]
> @otisg `sematext/agent:latest` is actually the stable image version.

So what is stable then?  An alias for latest?

File: docs/agents/sematext-agent/containers/installation.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+docker run -d  --restart always --privileged -P --name st-agent \
+-v /sys/kernel/debug:/sys/kernel/debug \
+-v /var/run/:/var/run/ \
+-v /proc:/host/proc:ro \
+-v /etc:/host/etc:ro \
+-v /sys:/host/sys:ro \
+-v /usr/lib:/host/usr/lib:ro \
+-e CONTAINER_TOKEN==<YOUR_DOCKER_APP_TOKEN_HERE> \
+-e INFRA_TOKEN=<YOUR_INFRA_APP_TOKEN_HERE> \
+-e REGION=US \
```
Comment (otisg) [inline docs/agents/sematext-agent/containers/installation.md]: How will one know that EU is another valid option?  One would have to _read the text_ below.  Would _\<US or EU\>_ make more sense here?

File: docs/agents/sematext-agent/containers/installation.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+```bash
+docker run -d  --restart always --privileged -P --name st-agent \
+-v /sys/kernel/debug:/sys/kernel/debug \
+-v /var/run/:/var/run/ \
+-v /proc:/host/proc:ro \
+-v /etc:/host/etc:ro \
+-v /sys:/host/sys:ro \
+-v /usr/lib:/host/usr/lib:ro \
+-e CONTAINER_TOKEN==<YOUR_DOCKER_APP_TOKEN_HERE> \
```
Comment (otisg) [inline docs/agents/sematext-agent/containers/installation.md]: == or =?

**Comment by rabbitstack** [comment]
> So what is stable then? An alias for latest?

I've only encountered "Stable" in the official Helm chart name.

File: docs/agents/sematext-agent/containers/installation.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+# docker-compose.yml
+version: '3'
+services:
+  sematext-agent:
+    image: 'sematext/agent:latest'
+    environment:
+      - affinity:container!=*sematext-agent*
+      - CONTAINER_TOKEN=<YOUR_DOCKER_APP_TOKEN_HERE>
+      - INFRA_TOKEN=<YOUR_INFRA_APP_TOKEN_HERE>
+      - REGION=US
```
Comment (otisg) [inline docs/agents/sematext-agent/containers/installation.md]: `<US or EU>`?

File: docs/agents/sematext-agent/containers/installation.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+--mount type=bind,src=/var/run,dst=/var/run/ \
+--mount type=bind,src=/usr/lib,dst=/host/usr/lib \
+--mount type=bind,src=/sys/kernel/debug,dst=/sys/kernel/debug \
+--mount type=bind,src=/proc,dst=/host/proc,readonly \
+--mount type=bind,src=/etc,dst=/host/etc,readonly \
+--mount type=bind,src=/sys,dst=/host/sys,readonly \
+-e NODE_NAME={{.Node.Hostname}} \
+-e INFRA_TOKEN=<YOUR_INFRA_APP_TOKEN_HERE> \
+-e CONTAINER_TOKEN=<YOUR_DOCKER_APP_TOKEN_HERE> \
+-e REGION=US \
```
Comment (otisg) [inline docs/agents/sematext-agent/containers/installation.md]: `<US or EU>`?

File: docs/agents/sematext-agent/kubernetes/installation.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+    metadata:
+      labels:
+        app: sematext-agent
+    spec:
+      serviceAccountName: sematext-agent
+      containers:
+      - name: agent
+        image: sematext/agent:latest
+        imagePullPolicy: Always
+        env:
```
Comment (otisg) [inline docs/agents/sematext-agent/kubernetes/installation.md]: No REGION needed?

File: docs/agents/sematext-agent/kubernetes/installation.md [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+
+This helm chart installs the Sematext Agent to all nodes in your cluster via `DaemonSet` resource.
+
+To install it run the following command:
+
+```bash
+helm install --name sematext-agent \
+  --set containerToken=<YOUR_DOCKER_APP_TOKEN_HERE> \
+  --set infraToken=<YOUR_INFRA_APP_TOKEN_HERE> \
+  --set region=US \
```
Comment (otisg) [inline docs/agents/sematext-agent/kubernetes/installation.md]: `<US or EU>`?

File: docs/agents/sematext-agent/os-metrics.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
+title: Operating system metrics
+description: Operating system metrics collected by Sematext Agent.
+
+The agent collects basic system metrics including memory usage, CPU utilization, disk I/O statistics, and more. Most metrics are directly pulled from the _procsfs_ every 10 seconds, although it is possible to alter the collection interval by modifying the `st-agent.yml` configuration file and specifying the period along with the time unit:
+
+```yaml
+# Metric collection interval for OS, network & container metrics
+interval: 20s
```
Comment (otisg) [inline docs/agents/sematext-agent/os-metrics.md]: Isn't it 10?  If we use 10 by default I think we should show 10 here so people don't think we use 20.

File: docs/containers/index.md [hunk 8]
```diff
@@ -1,17 +1,12 @@
...
-
 ## Container Logging
 
 The [Logagent Docker image](https://hub.docker.com/r/sematext/logagent/) is tailored for the log collection on container platforms. It runs as a tiny container on every Docker host and collects logs for all cluster nodes and their containers. It is open-source agent created by Sematext.
 
-Logagent detects known log formats and structures your logs for supported applications and structures logs automatically. You can configure the log parser for any application by providing your pattern definitions or by contributing to the open source pattern definitions in the [patterns.yml](https://github.com/sematext/logagent-js/blob/master/patterns.yml) file. 
+Logagent detects known log formats and structures your logs for supported applications and structures logs automatically. You can configure the log parser for any application by providing your pattern definitions or by contributing to the open source pattern definitions in the [patterns.yml](https://github.com/sematext/logagent-js/blob/master/patterns.yml) file.
```
Comment (otisg) [inline docs/containers/index.md]: Logagent detects known log formats and structures your logs for supported applications and structures logs automatically. => Logagent detects known log formats and structures your logs for supported applications automatically.

File: docs/monitoring/infrastructure.md [hunk 9]
```diff
@@ -15,23 +15,23 @@ A Golang-based [Sematext agent](../agents/sematext-agent) (included in the spm-c
 
 Infra App also includes data about your containerized environments. Installation instructions for Docker Monitoring also include an Infra App token. That way when installing Sematext Agent for Docker, an Infra App will be installed as well.
 
-Infra Apps are currently available to all users for free as a Beta feature.
+Infra Apps are currently available for all users for free as a Beta feature.
```
Comment (otisg) [inline docs/monitoring/infrastructure.md]: for all => to all

**Pull request merged**